src/Hipace.cpp

```diff
@@ -119,6 +119,15 @@ Hipace::Hipace () :
     pph.query("MG_tolerance_rel", m_MG_tolerance_rel);
     pph.query("MG_tolerance_abs", m_MG_tolerance_abs);
 
+    if (maxLevel() > 0) {
+        AMREX_ALWAYS_ASSERT(maxLevel() < 2);
+        amrex::Array<amrex::Real, AMREX_SPACEDIM> loc_array;
+        pph.get("fine_tag_lo", loc_array);
+        for (int idim=0; idim<AMREX_SPACEDIM; ++idim) fine_tag_lo[idim] = loc_array[idim];
```

Comment:
Would
```suggestion
        pph.getarr("patch_lo", fine_tag_lo);
```
work instead?